xml/System.Reflection/NullabilityInfoContext.xml

```diff
@@ -129,10 +132,11 @@
         <Parameter Name="propertyInfo" Type="System.Reflection.PropertyInfo" />
       </Parameters>
       <Docs>
-        <param name="propertyInfo">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="propertyInfo">The property for which the nullability information gets populated.</param>
+        <summary>Populates a <see cref="T:System.Reflection.NullabilityInfo" /> for the given <see cref="T:System.Reflection.PropertyInfo" />. If the <c>nullablePublicOnly</c> feature is set for an assembly, like it does in the .NET SDK, the private and/or internal member's nullability attributes are omitted, and the API will return the <see cref="F:System.Reflection.NullabilityState.Unknown"/> state.</summary>
+        <returns>A <see cref="T:System.Reflection.NullabilityInfo" /> instance.</returns>
```

Comment:
NIT: not sure these `<see cref="T:System.Reflection.NullabilityInfo" />` should be updated to `<xref:System.Reflection.NullabilityInfo>`

Reply:
No, the see cref is correct. xref is only for Markdown sections.